Decompiler/Options.cs

```diff
@@ -42,6 +42,26 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option("tool", DefaultValue = false,
+            HelpText = "Enable generic vpk dump tool for all supported resources.")]
+        public bool VPKTool { get; set; }
```

Comment:
What's the point of this option? If you wanted to keep gametracking compat, I suggested that removing that hardcoded code is fine as long as the same behaviour can be achieved using CLI only (extract + filter + dump).